runtime/gc_base/FinalizerSupport.cpp

```diff
@@ -553,6 +553,18 @@ static int J9THREAD_PROC FinalizeSlaveThread(void *arg)
 
 		fns->internalEnterVMFromJNI(env);
 		
+#if defined(J9VM_GC_DYNAMIC_CLASS_UNLOADING)
+		if(slaveData->mode != FINALIZE_SLAVE_MODE_CL_UNLOAD) {
```

Comment:
you can move opening "{" down outside "if defined". With this you can avoid "if defined" on closing bracket.